qiita_pet/templates/study_description_templates/raw_data_info.html

```diff
@@ -0,0 +1,56 @@
+<div>
+  <div class="panel-group" id="accordion-rd-info-{{rd_id}}-{{prep_template_id}}" role="tablist" aria-multiselectable="true">
+    <div class="panel panel-default">
+      <div class="panel-heading" role="tab" id="rd-info-{{rd_id}}-{{prep_template_id}}">
+        <h4 class="panel-title">
+          <a class="collapsed" data-toggle="collapse" data-parent="#accordion-rd-info-{{rd_id}}-{{prep_template_id}}" href="#collapse-rd-info-{{rd_id}}-{{prep_template_id}}" aria-expanded="false" aria-controls="collapse-rd-info-{{rd_id}}-{{prep_template_id}}">
+            Raw data - {{rd_filetype}} (ID: {{rd_id}})
```

Comment:
This really doesn't need to be an accordion, we can just show the files in a table by type.